client/src/API.ts

```diff
@@ -1,5 +1,5 @@
 import { mongoose } from '@typegoose/typegoose';
-import { KxDocument, PageRange } from './model';
+import { KxDocument, PageRange, Scale } from './model';
```

Comment:
perhaps it could be better to put the scale value on 1:N on the same row with the scale, I mean playing with the column width, but generally that's ok!

Reply:
Thank you for the suggestion! I’ll work on implementing and testing it.

Reply:
It does look better aesthetically, but the smaller form might impact usability.